pkg/cloud/gcp/actuators/util/gcp_uefi_disk_check_test.go

```diff
@@ -0,0 +1,206 @@
+package util_test
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	machinev1 "github.com/openshift/api/machine/v1beta1"
+	machinev1builder "github.com/openshift/cluster-api-actuator-pkg/testutils/resourcebuilder/machine/v1beta1"
+	computeservice "github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/services/compute"
+	"github.com/openshift/machine-api-provider-gcp/pkg/cloud/gcp/actuators/util"
+)
+
+var _ = Describe("IsUEFICompatible", func() {
+	var (
+		computeService      computeservice.GCPComputeService
+		providerSpecBuilder machinev1builder.GCPProviderSpecBuilder
+		providerSpec        *machinev1.GCPMachineProviderSpec
+
+		compatible bool
+		err        error
+	)
+
+	BeforeEach(func() {
+		_, computeService = computeservice.NewComputeServiceMock()
+		providerSpecBuilder = machinev1builder.GCPProviderSpec()
```

Comment:
You may as well move this into the table func right? And then you can get rid of all of the vars above?